packages/subgraph/tests/token-voting/token-voting.test.ts

```diff
@@ -208,31 +207,30 @@ test('Run Token Voting (handleVoteCast) mappings with mock event', () => {
     START_DATE,
     END_DATE,
     SNAPSHOT_BLOCK,
-    MIN_SUPPORT,
-    MIN_TURNOUT,
+    SUPPORT_THRESHOLD,
+    PARTICIPATION_THRESHOLD,
     VOTING_POWER,
-    '1',
+    '2',
+    '0',
     '0',
-    '1',
     actions
   );
   // create event
   let event2 = createNewVoteCastEvent(
     PROPOSAL_ID,
     ADDRESS_ONE,
-    '1', // abstain
+    '2', // yes
     '1',
```

Comment:
It needed to change because we use the `worstCaseSupport` quantity now for comparision so having 1 yes vote and 1 abstain vote wasn't enough to be sure that the proposal is executable.